test/mergeUnorderedImportTests.ts

```diff
@@ -5,6 +5,18 @@ import { mergeTests } from '../src/mergeTests';
 import { assert as Assert } from 'chai';
```

Comment:
Why do we need chai? Also I've been writing assert in lower case, although I'm not 100% sure.

```suggestion
import * as assert from 'assert';
```

See also https://github.com/diffblue/java-combiner-js/pull/31